code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -13,7 +13,7 @@
 	vital = 1
 	amputation_point = "spine"
 	gendered_icon = 1
-	cannot_amputate = 1
+	//cannot_amputate = 1
```

Comment:
If this isn't being used anymore then why not just remove it?

Reply:
Yeah, get rid of this, please.